lib/translations.json

```
@@ -3048,7 +3048,23 @@
     },
     {
       "key": "article/progress/backtotopbutton/title",
-      "value": "Nach oben"
+      "value": "Zum Anfang"
+    },
+    {
+      "key": "article/progress/time",
+      "value": "Ihre Leseposition {day} um {time} Uhr."
+    },
+    {
+      "key": "article/progress/time/today",
+      "value": "von heute"
+    },
+    {
+      "key": "article/progress/time/yesterday",
+      "value": "von gestern"
+    },
+    {
+      "key": "article/progress/time/other",
+      "value": "vom {date}"
```

```suggestion
      "value": "am {date}"
```

Schweizer nutzen "*am Tag.Monat.Jahr"

---

Merci vielmal. Isch jetz helvetisiert, au de Tescht.